lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -67,6 +68,7 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
             segment_extension: String.t(),
             header_naming_fun: (Track.t(), counter :: non_neg_integer() -> String.t()),
             segment_naming_fun: (Track.t() -> String.t()),
+            partial_naming_fun: (String.t(), String.t() -> String.t()),
```

Comment:
Wouldn't it make sense to generate the partial names out of `Track.t`, just like regular segments?

Reply:
I can do that but It will work only inside track module. I wouldn't be able to use it for manifest serialization to create a preload hint URI. So there would be two different functions or one but with an additional field. 
```
def partial_naming_fun(track, [preload_hint?: preload_hint?] \\ [preload_hint?: false]

Reply:
But segment_naming_fun doesn't have this argument as well 🤔

Reply:
because the preload hint is simply the next partial that is not yet produced so it doesn't make sense to have it in segment_naming_function.